AirCasting/MicrophoneSession/MicrophoneManager.swift

```diff
@@ -100,15 +112,22 @@ private extension MicrophoneManager {
         let power = recorder.averagePower(forChannel: 0)
         let decibels = Double(power + 90.0)
         let location = obtainCurrentLocation()
-        Log.debug("New mic measurement \(decibels) at \(String(describing: location))")
-        try measurementStreamStorage.addMeasurementValue(decibels, at: location, toStreamWithID: measurementStreamLocalID!)
+        
+        measurementStreamStorage.accessStorage { [self] storage in
```

Comment:
strong capturing `[self]` is redundant (seems like `self` is not used inside too)

Reply:
same as above :)